src/exportable/ExportableInstance.ts

```diff
@@ -1,8 +1,9 @@
 import { fshtypes } from 'fsh-sushi';
-import { Exportable, ExportableAssignmentRule, ExportableInsertRule } from '.';
+import { Exportable, ExportableAssignmentRule, ExportableInsertRule, ExportablePathRule } from '.';
 
 export class ExportableInstance extends fshtypes.Instance implements Exportable {
-  rules: (ExportableAssignmentRule | ExportableInsertRule)[];
+  //@ts-ignore
+  rules: (ExportableAssignmentRule | ExportableInsertRule | ExportablePathRule)[];
```

Comment:
Yeah, the fact that none of the SUSHI types accept `PathRule` made things tricky.  I think `@ts-ignore` is the way to go for now, but maybe we should consider updating SUSHI to allow `PathRule` on these. Since a `PathRule` is basically a no-op, I don't think it would do much harm to allow them and then ignore them if they _somehow_ ever made it into an instance in SUSHI.

Reply:
I don't think SUSHI even has a `PathRule` class to include. It's only in the grammar, so we would need to add a class that we don't really ever need, which feels a little weird.

Reply:
Lol true...

I don't love my options there and I don't know which one is weirder 😆

Reply:
I think I might actually have a use case for why we might _want_ to preserve `PathRule`s in SUSHI... I thought of it when reviewing the manual slice ordering PR in SUSHI.

Let's say that I have an array that is sliced -- but the slice requirements represent the whole value of the slice.  For example, in an `Observation` profile:
```
Profile: LabObservation
Parent: Observation
Description: "An Observation with category fixed to "
* category ^slicing.discriminator.type = #pattern
* category ^slicing.discriminator.path = "$this"
* category ^slicing.rules = #open
* category ^slicing.description = "Slice by category value"
* category ^slicing.ordered = false
* category contains lab 1..1
* category[lab] = $OBSCAT#laboratory
```

Now let's say I have an instance where I want the `laboratory` category to come first, and then a `blood-lab` category.  Theoretically, I should be able to do this:
```
* category[lab]
* category[+] = $CUSTOMSYSTEM#blood-lab
```

I don't need to assign `category[lab]` because the value is already fixed and will be populated for me. So I just use a `PathRule` to indicate the ordering I want.  Right now, I don't get what I want because the `PathRule` is stripped out in the importer.  But if it were preserved, then it would be considered when determining the ordering.

Similarly, if I have an optional slice with fixed values within it, maybe I want to indicate that the slice _should_ be exported in the instance by just including a `PathRule` to the slice. Right now, the `PathRule` would get stripped out in the exporter and the slice wouldn't be included at all.

So... maybe we should think more about supporting `PathRule`s all the way through the SUSHI process.

Reply:
So after further conversation, I think the approach is:
* Use `@ts-ignore` for now.
* Update SUSHI to support `PathRule` (because it's needed for other use cases there too).
* Then integrate it back into GoFSH, removing the `@ts-ignore`s.